### ISSUE TITLE
Translations inside JS files aren't being found when generating pot file

### ISSUE THREAD

**Issue by pattonwebz** [equalizedigital/accessibility-checker#991]
The translations included in JS files (other than frontend fixes bundle) aren't detected when a pot is generated due to terser renaming the translation functions. This should be fixed
